components/scream/tests/uncoupled/shoc/CMakeLists.txt

```diff
@@ -1,19 +1,48 @@
 include (ScreamUtils)
 
 # Create the test
+SET (TEST_LABELS "shoc;physics")
 set (NEED_LIBS shoc scream_control scream_share)
-CreateUnitTest(shoc_standalone "shoc_standalone.cpp" "${NEED_LIBS}" LABELS "shoc;physics")
+CreateUnitTest(shoc_standalone "shoc_standalone.cpp" "${NEED_LIBS}" LABELS ${TEST_LABELS}
+  MPI_RANKS 1 ${SCREAM_TEST_MAX_RANKS}
+  PROPERTIES FIXTURES_SETUP shoc_main
```

Comment:
Subjective: `shoc_main` (as well as `p3_main`. `spa_main`,... in other tests) is a somewhat overloaded string. I don't want other devs down the road to think this string is related to the funciton `shoc_main`. Perhaps we could use something like `generate_output_nc_files` or something quite obvious like that?